src/h3lib/lib/h3Index.c

```diff
@@ -266,6 +266,9 @@ H3Index H3_EXPORT(h3ToCenterChild)(H3Index h, int childRes) {
  */
 int H3_EXPORT(compact)(const H3Index* h3Set, H3Index* compactedSet,
                        const int numHexes) {
+    if (numHexes == 0) {
```

Comment:
Do we need similar handling on `uncompact`?

Reply:
`uncompact` and `maxUncompactSize` have for loops like `for (int i = 0; i < numHexes; i++)`, so they are already exiting when the number of hexagons is 0. I added tests in this PR to confirm this.